vmb/templates/matrimony/profile_edit.html

```diff
@@ -70,9 +70,7 @@ <h3>Drop files here or click to upload.</h3>
                     {% endfor %}
                 </div>
                 </div>
-                    {% if form.wizard %}
                     {% crispy form %}
```

Comment:
This commit solves the save button problem for non-wizard version of the photos page.